src/Kunstmaan/NodeSearchBundle/Helper/SearchRenderHelperInterface.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Kunstmaan\NodeSearchBundle\Helper;
+
+use Kunstmaan\NodeBundle\Entity\NodeTranslation;
+use Kunstmaan\PagePartBundle\Helper\HasPagePartsInterface;
+use Symfony\Component\Templating\EngineInterface;
+
+/**
+ * Interface SearchRenderHelperInterface.
+ *
+ * Implement this interface to define custom rendering of the search.
+ */
+interface SearchRenderHelperInterface
+{
+    /**
+     * Removes all HTML markup & decode HTML entities
+     *
+     * @param string $text
+     *
+     * @return string
+     */
+    public function removeHtml($text);
```

Comment:
Not sure if this has to be part of the interface, as the interface is a contract that the class is able to render the search page for indexing. So this method doesn't seem to be required in that proces, it would be a private method in the actual implementation IMO